src/tests/baseservices/exceptions/simple/ParallelCrash.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Nit: not necessary.

Reply:
Copy & paste - removed in 4th commit, thanks for spotting.